AmplifyPlugins/Analytics/Tests/AnalyticsHostApp/AWSPinpointAnalyticsPluginIntegrationTests/AWSPinpointAnalyticsPluginIntegrationTests.swift

```diff
@@ -18,14 +18,25 @@ import Network
 class AWSPinpointAnalyticsPluginIntergrationTests: XCTestCase {
 
     static let amplifyConfiguration = "testconfiguration/AWSPinpointAnalyticsPluginIntegrationTests-amplifyconfiguration"
+    static let amplifyOutputs = "testconfiguration/AWSPinpointAnalyticsPluginIntegrationTests-amplify_outputs"
     static let analyticsPluginKey = "awsPinpointAnalyticsPlugin"
     
+    var useGen2Configuration: Bool {
+        ProcessInfo.processInfo.arguments.contains("GEN2")
+    }
```

Comment:
i think that is only used by the unit test targets. the host apps consume the amplify products and `AmplifyTestCommon` is not one of them, unless I missed something here

Reply:
Keeping this "GEN2" check within the test seems reasonable since that's where the argument for the test plan is also configured. If we keep it in a common place, then of course there's some less flexibility because all test plans have to use the same argument, which is the case now but may not be the case in the future or all test targets across the categories